etcdctl/go.mod

```diff
@@ -30,7 +30,6 @@ replace (
 	go.etcd.io/etcd/pkg/v3 => ../pkg
 	go.etcd.io/etcd/raft/v3 => ../raft
 	go.etcd.io/etcd/server/v3 => ../server
-//	go.etcd.io/etcd/v3 => ../
```

Comment:
I can put this import in the replace block [currently][1] at the bottom of this file, if desired:

```
replace (
	go.etcd.io/etcd => ./FORBIDDEN_DEPENDENCY
	go.etcd.io/tests/v3 => ./FORBIDDEN_DEPENDENCY
)
```

[1]: https://github.com/etcd-io/etcd/blob/a3174d0f8ec6ec58827d7d86448bb4df08ae69e4/etcdctl/go.mod#L38-L41

Reply:
(Also worth noting, this change doesn't fall under the purview of my PR subject, if you'd like I can create a second PR or make a more all-encompassing PR subject.)

Reply:
Actually I would put this one /etcd/v3 to target to the ./FORBIDDEN_DEPENDENCY

Reply:
Done, thanks.